search/src/test/nodejs/search_combiner.test.js

```diff
@@ -70,14 +76,17 @@ describe('combine EM SearchResults', () => {
     };
 
     beforeEach(() => {
+        console.log('!!!!! 1');
```

Comment:
This looks temporary... ?

Reply:
it was - I missed it when I removed other debug statements